### PR TITLE
Updating versions for 1.7.1

### PR DESCRIPTION
**Pull request by annarev**

### CODE REVIEW

**Review by yifeif — APPROVED** [review]

**Comment by yifeif** [comment]
I think windows bazel build was setup after 1.7. We can ignore it.

**Review by av8ramit — APPROVED** [review]

File: tensorflow/docs_src/install/install_sources.md [hunk 1]
```diff
@@ -459,8 +459,8 @@ Stack Overflow and specify the `tensorflow` tag.
 **Linux**
 <table>
 <tr><th>Version:</th><th>CPU/GPU:</th><th>Python Version:</th><th>Compiler:</th><th>Build Tools:</th><th>cuDNN:</th><th>CUDA:</th></tr>
-<tr><td>tensorflow-1.7.0</td><td>CPU</td><td>2.7, 3.3-3.6</td><td>GCC 4.8</td><td>Bazel 0.10.0</td><td>N/A</td><td>N/A</td></tr>
```
Comment (av8ramit) [inline tensorflow/docs_src/install/install_sources.md]: Let's change these back to 1.7.0 for parity.

File: tensorflow/docs_src/install/install_sources.md [hunk 2]
```diff
@@ -480,7 +480,7 @@ Stack Overflow and specify the `tensorflow` tag.
 **Mac**
 <table>
 <tr><th>Version:</th><th>CPU/GPU:</th><th>Python Version:</th><th>Compiler:</th><th>Build Tools:</th><th>cuDNN:</th><th>CUDA:</th></tr>
-<tr><td>tensorflow-1.7.0</td><td>CPU</td><td>2.7, 3.3-3.6</td><td>Clang from xcode</td><td>Bazel 0.10.1</td><td>N/A</td><td>N/A</td></tr>
+<tr><td>tensorflow-1.7.1</td><td>CPU</td><td>2.7, 3.3-3.6</td><td>Clang from xcode</td><td>Bazel 0.10.1</td><td>N/A</td><td>N/A</td></tr>
```
Comment (av8ramit) [inline tensorflow/docs_src/install/install_sources.md]: Let's change these back to 1.7.0 for parity.

**Review by av8ramit — CHANGES_REQUESTED** [review]

**Comment by annarev** [comment]
I changed versions in install_sources.md tables back to 1.7.0.

**Review by gunan — APPROVED** [review]

**Pull request merged**